PluginDirectories/1/workingon.bundle/options.json

```diff
@@ -0,0 +1,9 @@
+{
+	"options": [
+		{
+			"text": "Your WorkingOn.co Token: ",
+			"type": "text",
+			"key": "token"
+		}
```

Comment:
It works for me! (if you make changes to a plugin, they aren't reloaded
automatically — you've got to restart Flashlight first)

On Wed, Mar 4, 2015 at 11:44 AM, Tobias O. <notifications@github.com> wrote:

> In PluginDirectories/1/workingon.bundle/options.json
> <https://github.com/nate-parrott/Flashlight/pull/355#discussion_r25789700>
> :
>
> > @@ -0,0 +1,9 @@
> > +{
> > +	"options": [
> > +		{
> > +			"text": "Your WorkingOn.co Token: ",
> > +			"type": "text",
> > +			"key": "token"
> > +		}
>
> https://www.workingon.co/tools
> that shows the token at the bottom - it's kinda obvious though, their page
> literally only has three useful pages - the main page where you have only
> an input, the activity feed, and the one with integrations from above.
> can you maybe test this thing on your machine? I really can't make it work
> inside flashlight itself - it just always gets overwritten by search
> results from spotlight.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nate-parrott/Flashlight/pull/355/files#r25789700>.
>